blockchair/test/balance.test.ts

```diff
@@ -0,0 +1,229 @@
+import { assert } from 'chai'
```

Comment:
These balance tests look like they could and should be reused for any "balance supported" adapter. 
Please make a specification and let's reuse it for all PoR adapters. The only thing to think about is where to put it 🧐